xapian-letor/api/Makefile.mk

```diff
@@ -1,9 +1,11 @@
 noinst_HEADERS +=\
-	api/featurevector_internal.h
+	api/featurevector_internal.h\
+	api/featurelist_internal.h
```

Comment:
These lists of source files should be sorted to make it easy to locate an entry, so you've added this in the wrong place.